.drone.star

```diff
@@ -273,6 +273,10 @@ def docker(config):
               'OWNCLOUD_DB_USERNAME': 'owncloud',
               'OWNCLOUD_DB_PASSWORD': 'owncloud',
               'OWNCLOUD_DB_NAME': 'owncloud',
+              # $config['integrity.excluded.files'] = explode(',', getenv('OWNCLOUD_INTEGRITY_EXCLUDED_FILES'));
+              'OWNCLOUD_INTEGRITY_EXCLUDED_FILES': 'something',
+              # $config['integrity.check.disabled'] = getenv('OWNCLOUD_INTEGRITY_CHECK_DISBLED') == 'true';
+              'OWNCLOUD_INTEGRITY_CHECK_DISABLED': 'true',
```

Comment:
This seems a good thing to do - this is the pipeline just for UI tests. The server of those pipelines will be "thrown away" at the end of the test run. So the setting will not end up in the published images.

Reply:
We will use an occ command.
I am not eager to add this variable to the official base image.

Reply:
I saw your commit in #204 - that is easy and good, the setting only happens directly in the UI pipelines.

Reply:
ok. we agree that disabling the integrity checker is a good thing here.
Now we need to find a syntax that allows us to do that. 
the syntax here does not wor. The variable OWNCLOUD_INTEGRITY_CHECK_DISABLED is my private idea, it does not exist in the base images.